<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAnimeListDialog</class>
 <widget class="QDialog" name="MyAnimeListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Importuj listę seriali z pliku XML</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="instructionsLabel">
     <property name="text">
      <string>Wybierz plik XML z listą seriali do importowania:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filePickerLayout">
     <item>
      <widget class="QLineEdit" name="filePathInput">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Ścieżka do pliku XML</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Przeglądaj...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>Importuj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
